emalloc.c

```diff
@@ -451,13 +451,10 @@ static void* alloc_from_meta(size_t bsize)
     return block_to_payload(b);
 }
 
-int emalloc_init()
+int emalloc_init_reserve(size_t init_size)
```

Comment:
Who will call this `emalloc_init_reserve()` then, the RTS? It seems to turn this API to public -- which I think is not expected?

Reply:
tRTS of Intel SGX SDK will call it at enclave initialization time, so it need to be exposed. Other customized SDKs may just ignore it.